pkg/cmd/root.go

```diff
@@ -71,3 +71,10 @@ func errorIfFileDoesNotExist(val *types.CmdValues) {
 		os.Exit(1)
 	}
 }
+
+func setDefaultComposeFile(val *types.CmdValues) {
+	val.Files = []string{"docker-compose.yml"}
+	if utils.CheckIfFileExists(val.Files) != nil {
+		val.Files = []string{"docker-compose.yaml"}
```

Comment:
This will make `docker-compose.yaml` and when this file is not found it will throw error for `docker-compose.yaml`.